api/external/applications/applications.proto

```diff
@@ -9,9 +9,6 @@ import "protoc-gen-swagger/options/annotations.proto";
 // TODO @afiune Move or re-create the common messages inside the api/ folder
 import "api/external/common/version/version.proto";
 
```

Comment:
Totally by chance, this first file shows all 3 variations of things that were stripped via `sed`.